system/Common.php

```diff
@@ -414,10 +414,11 @@ function env(string $key, $default = null)
      * If $data is an array, then it loops over it, escaping each
      * 'value' of the key/value pairs.
      *
-     * Valid context values: html, js, css, url, attr, raw
-     *
      * @param array|string $data
-     * @param string       $encoding
+     * @param string       $context  Valid context values: html, js, css, url, attr, raw
```

Comment:
Let's make these literals, at least for PHPStan:
```suggestion
     * @param 'html'|'js'|'css'|'url'|'attr'|'raw' $context
```